.storybook/preview.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {Theme} from '@twilio-paste/theme';
+
+export const parameters = {
+  actions: {argTypesRegex: '^on[A-Z].*'},
+};
+
+export const decorators = [
+  Story => (
+    <Theme.Provider theme="default">
```

Comment:
We lose the theme switcher for now because we need to migrate the addon.

Reply:
I wrote a theme switcher for the SendGrid storybook, it was pretty easy to create.

```typescript
import React from "react";
import { Theme } from "@twilio-paste/core/theme";

export const globalTypes = {
  theme: {
    name: "Paste Theme",
    description: "Global theme for components",
    defaultValue: "console", // console looks more like sendgrid than "sendgrid"
    toolbar: {
      // All available icons
      // https://github.com/storybookjs/storybook/blob/master/lib/components/src/icon/icons.tsx
      icon: "paintbrush",
      // array of plain string values or MenuItem shape (see below)
      items: ["sendgrid", "console", "flex"],
    },
  },
};

export const parameters = {
  actions: { argTypesRegex: "^on.*" },
  rootStyles: {
    height: "100vh",
    padding: "60px",
    backgroundColor: "white",
  },
};

export const decorators = [
  (Story, context) => {
    const theme = context.globals.theme;

    return (
      <Theme.Provider theme={theme}>
        <div style={context.parameters.rootStyles}>
          <Story />
        </div>
      </Theme.Provider>
    );
  },
];

```